includes/polling/core.inc.php

```diff
@@ -79,14 +79,18 @@
     }
 }
 
-if ($poll_device['sysLocation'] && $device['location'] != $poll_device['sysLocation'] && $device['override_sysLocation'] == 0) {
-    $update_array['location'] = $poll_device['sysLocation'];
-    $device['location']       = $poll_device['sysLocation'];
-    log_event('Location -> ' . $poll_device['sysLocation'], $device, 'system', 3);
-}
+if ($device['override_sysLocation'] == 0 && $poll_device['sysLocation']) {
```

Comment:
This now stops you overriding the devices location but still have it geo lookup what that value is.